content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Scale Sensu Go with Enterprise Datastore"
+linkTitle: "Routing Alerts with Filters"
+description: ""
+weight: 39
+version: "5.10"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.10:
+    parent: guides
+---
+
+Sensu Go's licensed-tier datastore feature enables scaling your monitoring to many thousands of events per second.
+
+- [Why use the Enterprise datastore?](#why-enterprise-datastore)
+- [Prerequisites](#prerequisites)
+- [Configure Postgres](#configure-postgres-for-sensu)
+- [Configure Sensu](#configure-sensu-for-postgres)
+- [Revert to built-in datastore](#revert-to-builtin-datastore)
+
+## Why use the Enterprise datastore? {#why-enterprise-datastore}
+
+For each unique entity/check pair, Sensu records the latest event object in its datastore. By default, the embedded etcd datastore is used for event storage. The embedded etcd datastore helps you get started with Sensu, but as the number of entities and checks in your Sensu implementation grows, so does the rate of events being written to the datastore. In a clustered deployment of etcd, whether embedded or external to Sensu, each event received by a member of the cluster must be replicated to other members, increasing network and disk IO utilization. 
+
+Our team has documented configuration and testing of Sensu running on bare metal infrastructure in the [sensu/sensu-perf][2] project. This configuration comfortably handled 12,000 Sensu agent connections (and their keepalives) and processed more than 8,500 events per second. 
+
+This rate of events should be more than sufficient for many installations but assumes an ideal scenario where Sensu backend nodes use direct-attached, dedicated NVMe storage and are connected to a dedicated LAN. Deployments on public cloud providers are not likely to achieve similar results due to sharing both disk and network bandwidth with other tenants. Following the cloud provider's recommended practices may also become a factor because many operators are inclined to deploy a cluster across multiple availability zones. In such a deployment cluster, communication happens over shared WAN links, which are subject to uncontrolled variability in throughput and latency.
```

Comment:
```suggestion
This rate of events should be more than sufficient for many installations but assumes an ideal scenario where Sensu backend nodes use direct-attached, dedicated non-volatile memory express (NVMe) storage and are connected to a dedicated LAN. Deployments on public cloud providers are not likely to achieve similar results due to sharing both disk and network bandwidth with other tenants. Following the cloud provider's recommended practices may also become a factor because many operators are inclined to deploy a cluster across multiple availability zones. In such a deployment cluster, communication happens over shared WAN links, which are subject to uncontrolled variability in throughput and latency.
```